io/driver/module_unload_load.py

```diff
@@ -98,6 +98,7 @@ def module_load_unload(self, module_list):
                     for mod in sub_mod.split(' '):
                         if mod == 'multipath':
                             if self.flush_mpath(mdl) is False:
+                                self.error("failed to flush the multipath")
                                 self.error_modules.append(mdl)
```

Comment:
if the self.error is called,, it will not append in next line.. meaning the test exits

Reply:
i say use self.fail instead of self. error

Reply:
ahh, that was just a pointer I have added to debug exact issue.
I have removed it. As we are capturing the module in a list and validating at the end of the test.
Hence it is not required here.
just removed that line. please re-check